powershell/cmdlets/namespace.ts

```diff
@@ -56,7 +56,7 @@ export class NewCmdletNamespace extends Namespace {
         continue;
       }
       // skip-for-time-being
-      this.addClass(await new NewCmdletClass(this, operation, this.state.path('commands', 'operations', index)).init());
+      // this.addClass(await new NewCmdletClass(this, operation, this.state.path('commands', 'operations', index)).init());
```

Comment:
Any reason for commenting out this line

Reply:
Consider powershell-v2 has some issue

Reply:
Fixed. And also change pipeline to use Powershellv2. Let's confirm whether it can pass the test ;-)